operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -3288,6 +3290,9 @@ gateway:
         error_log  /dev/stderr {{ .Values.gateway.nginx.config.errorLogLevel }};
         pid        /tmp/nginx.pid;
         worker_rlimit_nofile 8192;
+        {{- if .Values.gateway.nginx.config.clientMaxBodySize }}
+        client_max_body_size {{ .Values.gateway.nginx.config.clientMaxBodySize }};
+        {{- end }}
```

Comment:
We're getting a problem with this new change:
```
nginx: [emerg] "client_max_body_size" directive is not allowed here in /etc/nginx/nginx.conf:5
```
This directive is not allowed in this context. Could we get this removed from here?